libr/asm/arch/ppc/gnu/ppc-dis.c

```diff
@@ -76,7 +76,7 @@ powerpc_init_dialect (struct disassemble_info *info)
   ppc_cpu_t dialect = PPC_OPCODE_PPC;
   struct dis_private *priv = calloc (sizeof (*priv), 1);
 
-  if (priv == NULL)
```

Comment:
this is code from GNU, ,so maybe is better to avoid touching that to make syncing with latest binutils easier